apps/builder/app/builder/shared/binding-popover.tsx

```diff
@@ -165,6 +183,17 @@ const BindingPanel = ({
           })}
         </CssValueListArrowFocus>
       </Box>
+      <Flex gap="1" css={{ px: theme.spacing[9], py: theme.spacing[5] }}>
+        <Text variant="labelsSentenceCase">Expression Editor</Text>
+        <Tooltip
+          variant="wrapped"
+          content={
+            "Compose variables, do math, write conditionals. Example: VariableA + VariableB > 10 ? true : false"
```

Comment:
We dont support ternaries. Only && and ||

Reply:
oh strange, I thought this would be the most interesting part in the expressions

Reply:
fixed